cmd/generate-config/config/config-openapi-spec.json

```diff
@@ -87,6 +88,11 @@
               "default": "InterNamespaceAllowed"
             }
           }
+        },
+        "status": {
+          "description": "Default router status, can be Enabled or Disabled.",
+          "type": "string",
+          "default": "Enabled"
```

Comment:
Can we change `status` to `enabled` and using boolean type `true/false` here? status might be confused and it doesn't sound like a config.

Reply:
This is done purposefully - it allows extending it in the future rather being limited by two values: true or false.

On the other hand I see in the enhancement it was changed to `Managed`/`Removed` (https://github.com/openshift/enhancements/pull/1548#discussion_r1531085819) so `Enabled` is no longer valid and should be changed

Reply:
Just updated to match the latest values.